sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -48,6 +51,26 @@ export class EducationProgramController {
     }));
   }
 
+  @Get(":programId")
+  async get(
+    @Param("programId") programId: number,
+  ): Promise<EducationProgramDto> {
+    const educationProgram = await this.programService.getLocationPrograms(
```

Comment:
We need to check if the institution requesting the program id has access to it.

Reply:
Done